model/event/task_event.go

```diff
@@ -126,6 +128,11 @@ func LogTaskRestarted(taskId string, execution int, userId string) {
 	logTaskEvent(taskId, TaskRestarted, TaskEventData{Execution: execution, UserId: userId})
 }
 
+// TODO: should we also log what task blocked it? (Would only want to log it then if it's blocked for the first time I think)
```

Comment:
I think it would be nice to log it!

Reply:
Sorry this is actually an old todo--I decided not to do this, because we don't display this on the event log page. I think this would clutter that page since blocked information is already easily available on the main page, and would be confusing if we didn't log every time and then also log when something was unblocked.